### PR TITLE
Add some speedrun mods

### PR DESCRIPTION
**Pull request by HowardZHY**
from https://www.minecraftspeedrunning.com/public-resources/mods

### CODE REVIEW

**Comment by BleachDev** [comment]
Maybe change the name of `Atum (Random and set seed autoresetter)` to `Atum (Autoresetter)` or just `Atum` since the name is very long compared to the other mods on the list

File: res/mods.js [hunk 1]
```diff
@@ -126,7 +142,7 @@ const mods = [
             "modrinth": "https://modrinth.com/mod/fabricated-forge",
         },
         "working": true,
-        "versions": [ "1.4.2", "1.4.1", "1.4", "1.3.2" ]
+        "versions": [ "1.4.7", "1.3.2" ]
```
Comment (arthurbambou) [inline res/mods.js:145]: why do you remove minor versions?

**Review by arthurbambou — CHANGES_REQUESTED** [review]

File: res/mods.js (hunk 1 above)
Comment (HowardZHY) [reply res/mods.js:145]: Minor versions are unnecessary. 
I think the way to show versions needs to be improved, like using version range eg. 1.3.2-1.4.7 or 1.4.X.

Comment (arthurbambou) [reply res/mods.js:145]: The current setup is not made to work this way tho, we can do a refactor later in another pr, for now tho just don't remove those stuff.

Comment (HowardZHY) [reply res/mods.js]: maybe filter with 1.3, 1.4, 1.5... instead of full version is an idea

**Review by arthurbambou — APPROVED** [review]

**Review by halotroop2288 — APPROVED** [review]

**Pull request merged**